mscore/plugin/api/qmlpluginapi.cpp

```diff
@@ -177,6 +177,19 @@ Element* PluginAPI::newElement(int elementType)
       return wrap(e, Ownership::PLUGIN);
       }
 
+//---------------------------------------------------------
+//   removeElement
+///   Disposes of an Element and its children.
+///   \param Element type.
+///   \since MuseScore 3.3
+//---------------------------------------------------------
+
+void PluginAPI::removeElement(Ms::PluginAPI::Element* wrapped)
+      {
+      Ms::Score* score = msc()->currentScore();
+      score->undoRemoveElement(wrapped->element());
```

Comment:
As I denoted in [this comment](https://musescore.org/en/node/291790#comment-935704), it may be safer to use `deleteItem()` here